b-real-time-htb.js

```diff
@@ -150,12 +157,27 @@ function BRealTimeHtb(configs) {
          */
 
         /* PUT CODE HERE */
+        
+        var __tags = [];
+        for(var i=0;i<returnParcels.length;i++) {
+          let returnParcel = returnParcels[i], tag = {};
+          tag.sizes = returnParcel.xSlotRef.sizes.map(function(obj) {
+            return {width:obj[0], height:obj[1]};
+          });
+          tag.id = returnParcel.xSlotRef.placementId;
+          tag.primary_size = tag.sizes[0];
+          tag.allow_smaller_sizes = false;
+          tag.prebid = true;
+          tag.disable_psa = true;
+          tag.uuid = System.generateUniqueId();
```

Comment:
Yes, my understanding is the UUID generated here is what will be passed in a 'live' environment, and if so that UUID should be passed back in the response - the UUID in the mock is so the tests will pass given they aren't hitting a live endpoint